pkg/piperutils/slices_test.go

```diff
@@ -133,3 +133,11 @@ func TestSplitTrimAndDeDup(t *testing.T) {
 		assert.Equal(t, []string{"a", "a", "b", "c", "d", "e", "f", "f"}, s)
 	})
 }
+
+func TestUniqueStrings(t *testing.T) {
+
+	unique := UniqueStrings([]string{"abc", "xyz", "123", "abc"})
```

Comment:
The sorting order should be clarified

Reply:
Not part of this PR anymore. Was merged with another PR. This PR depended on the other one. Nevertheless we should add this with another PR.